docs/main/concepts/data-engines/replicated-storage.md

```diff
@@ -28,9 +28,9 @@ In contrast, traditional pre-CNS shared storage systems are known to introduce o
 Although Replicated Storage utilizes NVMe-oF, it does not impose any requirements for the use of NVMe devices or cloud volumes.
 :::
 
-## Quickstart Guides
+## Quickstart Guide
 
-OpenEBS provides Local Volume that can be used to provide locally mounted storage to Kubernetes Stateful workloads. Refer to the [Quickstart Guide](../../quickstart-guide/) for more information.
+OpenEBS provides Local Volume that can be used to provide locally mounted storage to Kubernetes Stateful workloads. Refer to the [Quickstart Guide](../../quickstart-guide/installation.md) for more information.
```

Comment:
This seems not relevant here. It is mentioned local storage when the doc is about replicated storage?